locale/fi/LC_MESSAGES/django.po

```diff
@@ -4091,7 +4126,7 @@ msgstr "Pisteet"
 
 #: exercise/templates/exercise/_points_progress.html
 msgid "POINTS_TO_PASS"
-msgstr "Läpäisyyn vaadittavat pisteet"
+msgstr "Vaaditut pisteet"
```

Comment:
Why was this changed? I like the previous message because there are other messages like `läpäisyyn vaadittavat tehtävät`, and `läpäisyyn vaadittavat moduulit`, which it matches with.

Reply:
The original message flowed over the tooltip which looked particularly nasty now that there are three things on it with the points goal. So in production it is currently like this 
![image](https://github.com/user-attachments/assets/8d2485e6-efc2-4268-9535-2c4bf4f57b43).
Do you think this is ok as a shorter version or is the longer version preferrable even with the overflowing?

Reply:
Okay, I see the problem. You can keep the change.